crates/sui-types/src/committee.rs

```diff
@@ -284,6 +285,24 @@ impl Committee {
             .binary_search_by_key(name, |(a, _)| *a)
             .is_ok()
     }
+
+    /*
+    Testing-only methods
+     */
```

Comment:
nit: using block comments like this make it difficult to grep for code in the code base and know if something is live or commented out.